ParisBNF/abb/BNFabb94.xml

```diff
@@ -0,0 +1,324 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb94" xml:lang="en" type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Gadla ʾAbunāfǝr, Gadla Kiros, homilies on Melchizedek, Dǝrsana ʾAfnin</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MKr"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <editionStmt>
+                <p/>
+            </editionStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas:  Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                        <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b52520186b">BnF Éthiopien d'Abbadie 94</idno>
+                          </msIdentifier>
+
+                          <physDesc>
+                              <objectDesc form="Codex">
+                                  <supportDesc>
+                                      <support>
+                                 <material key="parchment">Parchment</material>
+                                         </support>
+                                      <extent>
+                                          <measure unit="leaf">62</measure>
+                                          <measure unit="leaf" type="blank"/>
+                                          <locus target="#1r #1v #32v #33r #33v #55v #56r #63r #63v"/>
```

Comment:
Hi Marcin, thank you for your work! I am now checking what you noted and see the following:
the total number of folia is 64, I think we put it as <measure unit="leaf" quantity="64">1+62+1</measure>
Then your following description is not correct, because f. 1r and 1v, according to the manuscript foliation is not blank. You need to assign different numbers to these endleaves. E.g. for d'Abbadie 106 I put it like this: 
       <measure unit="leaf" type="blank">2
                                        <locus target="#ir #ii  #58v"></locus></measure>
                                    <note>The first two leaves are not numbered,
                                        and have been assigned here the numbers <locus target="#i #ii"/>.
                                    </note>. 
If you want, you can check d'Abbadie 106. I worked on it with Denis and then everyone commented on it, so it is one of the (relatively) good records.